### PR TITLE
Adding note on performance_schema_max_digest_length

### PR DESCRIPTION
**Pull request by nethalo**

### CODE REVIEW

File: docs/setting-up/client/mysql.md [hunk 1]
```diff
@@ -206,6 +206,9 @@ To use *Performance Schema*, set the variables below.
 | [`performance-schema-consumer-statements-digest`][perfschema-consumer-statements-digest]   | `ON`               | Configures the `statements-digest` consumer.
 | [`innodb_monitor_enable`][sysvar_innodb_monitor_enable]                                    | all                | Enables InnoDB metrics counters.
 
+!!! caution alert alert-warning "Important"
+    When dealing with long queries, increasing the value of the variable [performance_schema_max_digest_length](https://dev.mysql.com/doc/refman/8.0/en/performance-schema-system-variables.html#sysvar_performance_schema_max_digest_length) will avoid having query examples truncated.
```
[the remark this answers is not in the document]
Comment (nethalo) [reply docs/setting-up/client/mysql.md:210]: But that is a pmm-admin config parameter and not a MySQL variable. I'd defer that to be added by the doc team

[the remark this answers is not in the document]
Comment (BupycHuk) [reply docs/setting-up/client/mysql.md:210]: than it won't be true in context of PMM. @rnovikovP PTAL ^

**Review by BupycHuk — CHANGES_REQUESTED** [review]

File: docs/setting-up/client/mysql.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (rnovikovP) [reply docs/setting-up/client/mysql.md:210]: As I understand, To have queries with full length it's 2 parameters need to be tuned: 
1) MySQL perf schema ^ 
2) PMM flag for the length 

If both of them are aligned - then the user will see the full query (most likely .. still can be a problem with veeeeryyyy long queries )

[the remark this answers is not in the document]
Comment (nethalo) [reply docs/setting-up/client/mysql.md:210]: This is a PR for a note explaining a Performance Schema modification. Please make another one for non-related information (non-mysql modification) @rnovikovP

**Review by BupycHuk — APPROVED** [review]

**Pull request merged**